### PR TITLE
refactor: rename packageTree to packageGraph

### PR DESCRIPTION
**Pull request by lee-tammy**

### CODE REVIEW

File: src/analysis-util.ts [hunk 1]
```diff
@@ -31,11 +31,11 @@ export enum IdType {
  * they are located in
  *
  * @param id the identifier that is being searched for
- * @param tree abstract syntax tree
+ * @param acornTree abstract syntax graph
```
Comment (jplaisted) [inline src/analysis-util.ts]: s/graph/tree/

**Review by jplaisted — APPROVED** [review]
Any particular reason for this change? I thought Tree was fine. Though I think Graph is fine too.

**Comment by lee-tammy** [comment]
We've been meaning to change it to graph because a node in the package**Thing** can have more than one parent, which makes it not a tree.

File: src/analysis-util.ts (hunk 1 above)
Comment (lee-tammy) [reply src/analysis-util.ts]: It should say tree.

Comment (jplaisted) [reply src/analysis-util.ts]: Right. Your variable name is `acornTree` but the description says `abstract syntax GRAPH`

s/a/b/ is short hand for "substitute a with b"

**Review by kjin — APPROVED** [review]
Looks good once John's comment is addressed.

File: src/analysis-util.ts (hunk 1 above)
Comment (lee-tammy) [reply src/analysis-util.ts]: I pushed my changes to the wrong branch :(

Comment (kjin) [reply src/analysis-util.ts]: 😅

**Pull request merged**